### PR TITLE
Standardize setup and build files.

### PR DESCRIPTION
**Pull request by Micky-G**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
-#### The command below does not need to be run again, this was used by me to setup the project.
-`ros2 pkg create --build-type ament_python cv_basics --dependencies rclpy image_transport cv_bridge sensor_msgs std_msgs python3-opencv`
+## SimpleTracker-ros2-ws
 
 ### The commands below should be used to get this ROS package up and running in VSCode and using the ROS dev container
 
-#### The following 2 ROS dependency commands should be included in the Dockerfile so that they don't have to be run manually, but I don't know how to do that yet
+#### Setup 
+Run the setup script to setup dev container
```
Comment (matt-ashman) [inline README.md:6]: Nice.

File: src/simple_tracker_detection_provider/simple_tracker_detection_provider/bgs_detector_node.py [hunk 2]
```diff
@@ -28,9 +28,9 @@ def __init__(self, subscriber_qos_profile: QoSProfile, publisher_qos_profile: Qo
 
     # setup services, publishers and subscribers
     self.sub_masked_background_frame = self.create_subscription(Frame, 'sky360/frames/masked_background/v1', 
-      self.masked_background_frame_callback, subscriber_qos_profile)
-    self.pub_key_points = self.create_publisher(KeyPointArray, 'sky360/detector/bgs/key_points/v1', publisher_qos_profile)
-    self.pub_bounding_boxes = self.create_publisher(BoundingBoxArray, 'sky360/detector/bgs/bounding_boxes/v1', publisher_qos_profile)   
+      self.masked_background_frame_callback, 10)#, subscriber_qos_profile)
+    self.pub_key_points = self.create_publisher(KeyPointArray, 'sky360/detector/bgs/key_points/v1', 10)#, publisher_qos_profile)
```
Comment (matt-ashman) [inline src/simple_tracker_detection_provider/simple_tracker_detection_provider/bgs_detector_node.py:32]: Should there be a named constant for the integer?

File: src/simple_tracker_configuration/simple_tracker_configuration/configuration_node.py [hunk 3]
```diff
@@ -35,8 +35,7 @@ def __init__(self, publisher_qos_profile: QoSProfile):
             self.get_config_array_callback)
         self.config_change_service = self.create_service(ConfigEntryUpdate, 'sky360/config/entry/update/v1', 
             self.get_config_update_callback)
-        self.config_change_publisher = self.create_publisher(ConfigEntryUpdatedArray, 'sky360/config/updated/v1', 
-            publisher_qos_profile)
+        self.config_change_publisher = self.create_publisher(ConfigEntryUpdatedArray, 'sky360/config/updated/v1', 10)#, publisher_qos_profile)
```
Comment (matt-ashman) [inline src/simple_tracker_configuration/simple_tracker_configuration/configuration_node.py:38]: Same here. If you want to change from `10` in the future that's a lot of typing.

Better to use a named constant.

**Review by matt-ashman — APPROVED** [review]

File: src/simple_tracker_detection_provider/simple_tracker_detection_provider/bgs_detector_node.py (hunk 2 above)
Comment (Micky-G) [reply src/simple_tracker_detection_provider/simple_tracker_detection_provider/bgs_detector_node.py:32]: Well spotted, so this is a shortcut way of defining some sort of QOS i.e. queue length is 10 messages in length. This simple QOS is temporary as I want to do proper QOS job (as I had in place prior to refactoring message types), so at the moment I just don't want to over complicate while I standardise message types. As soon as I am done with standardising message types I will provide an implementation for QOS.

File: src/simple_tracker_configuration/simple_tracker_configuration/configuration_node.py (hunk 3 above)
Comment (Micky-G) [reply src/simple_tracker_configuration/simple_tracker_configuration/configuration_node.py:38]: Same comment as above

**Pull request merged**